database-seed.sql

```diff
@@ -1,11 +1,49 @@
-CREATE TABLE testSeed
+CREATE TABLE users
 (
-    id SERIAL,
-    name text,
-    CONSTRAINT testSeed_pkey PRIMARY KEY (id)
+    user_id SERIAL PRIMARY KEY,
+    password VARCHAR (255) NOT NULL,
+    username VARCHAR (255) UNIQUE NOT NULL,
+    created_at TIMESTAMP DEFAULT NOW(),
+    deleted_at TIMESTAMP
 );
 
-INSERT INTO testSeed(name) VALUES
- ('Seed1'),
- ('Seed2'),
- ('Seed3');
\ No newline at end of file
+CREATE TABLE roles
+(
+    role_id SERIAL PRIMARY KEY,
+    role_name VARCHAR (50) UNIQUE NOT NULL
+);
+
+CREATE TABLE user_roles
+(
+    user_id INT NOT NULL,
+    role_id INT NOT NULL,
+    PRIMARY KEY (user_id, role_id),
+    FOREIGN KEY (role_id)
+      REFERENCES roles (role_id),
+    FOREIGN KEY (user_id)
+      REFERENCES users (user_id)
+);
+
+CREATE TABLE user_tokens
+(
+    user_id INT NOT NULL,
+    token TEXT DEFAULT '',
+    PRIMARY KEY (user_id),
+    FOREIGN KEY (user_id)
+      REFERENCES users (user_id)
+);
+
+INSERT INTO roles(role_name) VALUES ('admin');
+INSERT INTO roles(role_name) VALUES ('user');
+
+INSERT INTO users(password, username) VALUES ('$2a$12$Ppm4DOlHGXg.Svv.MS9LZeyPqPn7h2QYigrZ.6mlV..JW9R1h6Q3.', 'admin@test.com');
+INSERT INTO users(password, username) VALUES ('$2a$12$Ppm4DOlHGXg.Svv.MS9LZeyPqPn7h2QYigrZ.6mlV..JW9R1h6Q3.', 'user@test.com');
+INSERT INTO users(password, username, deleted_at) VALUES ('$2a$12$Ppm4DOlHGXg.Svv.MS9LZeyPqPn7h2QYigrZ.6mlV..JW9R1h6Q3.', 'deleted@test.com', NOW());
+
+INSERT INTO user_roles(user_id, role_id) VALUES (1,1);
+INSERT INTO user_roles(user_id, role_id) VALUES (1,2);
+INSERT INTO user_roles(user_id, role_id) VALUES (2,2);
+
+INSERT INTO user_tokens(user_id) VALUES (1);
```

Comment:
Overcomplicated from my side. I created table user_tokens where I stored user_id and actually generated for user token. I did it so that when creating a user, an entry with an empty token will be added. 
This is what you see. For the seed, I created it from the postgresql query line.

I've simplified it now. The currently used token is written to the token column in the table with users. It's clearer, simpler. It's hard to say why I chose that overcomplicated version before; P